library/Zend/Form/View/Helper/FormRow.php

```diff
@@ -213,7 +213,13 @@ public function render(ElementInterface $element)
                     $labelOpen = $labelClose = $label = '';
                 }
 
-                switch ($this->labelPosition) {
+                $labelPosition = $this->labelPosition;
```

Comment:
This line caused two test failures. Essentially, it was overriding lines 128-130, which were setting the value of `$labelPosition` to the instance property if it had not been passed. As such, I removed it, as the value will be properly initialized by this point already.